packages/react-reconciler/src/__tests__/ReactSuspensePlaceholder-test.internal.js

```diff
@@ -8,9 +8,9 @@
  * @jest-environment node
  */
 
-runPlaceholderTests('ReactSuspensePlaceholder (mutation)', () =>
-  require('react-noop-renderer'),
-);
+// runPlaceholderTests('ReactSuspensePlaceholder (mutation)', () =>
+//   require('react-noop-renderer'),
+// );
```

Comment:
Why disable these?

Reply:
Commented it out while debugging, meant to add back. Although now I just noticed... I screwed up when I converted this to use the test renderer. It's not actually testing anything in concurrent mode anymore :facepalm:

Reply:
I'll fix that in a separate PR since it's unrelated